resources/downloads.md

```diff
@@ -32,7 +33,9 @@ This is a fully configured image for the Raspberry Pi 3B computer to be used as
 ## QGroundControl
 
 <i class="fa fa-download" aria-hidden="true"></i> [Windows](https://s3.amazonaws.com/downloads.bluerobotics.com/QGC/latest/QGroundControl-installer.exe) (.exe)
+
```

Comment:
You should not have whitespace changes unrelated to the commit. If these changes are intentional, then they should go in a separate commit with a message like `fix whitespace`.

The easiest way to get only the correct changes in each/either commit is `git add -p`. https://nuclearsquid.com/writings/git-add/